src/table/table.scss

```diff
@@ -4,383 +4,423 @@
 @import '../icon/index';
 @import './variables';
 
-@mixin iui-tables-table {
+@mixin iui-table {
   @include iui-reset;
-
   @include iui-font-family;
   display: flex;
   flex-direction: column;
-  padding: 0;
 
   * {
     box-sizing: border-box;
   }
 
-  > .iui-tables-body {
-    width: 100%;
-    @include themed {
-      background-color: t(iui-color-background-1);
-    }
+  .iui-table-header {
+    @include iui-table-header;
   }
 
-  &.iui-zebra-striping > .iui-tables-body > .iui-tables-row:nth-child(even) {
+  .iui-table-body {
+    @include iui-table-body;
+  }
+
+  .iui-cell {
+    @include iui-table-cell;
+  }
+
+  &.iui-zebra-striping > .iui-table-body > .iui-row:nth-child(even):not(.iui-selected) {
     @include themed {
       background: rgba(t(iui-color-foreground-body-rgb), 0.02);
     }
   }
 
-  &.iui-tables-fixed > .iui-tables-body {
-    max-height: 300px;
+  &.iui-fixed > .iui-table-body {
+    max-height: $iui-baseline * 27;
     overflow-x: hidden;
     overflow-y: scroll;
     overflow: overlay;
   }
+}
 
-  .iui-select .iui-select-button {
-    background-color: transparent;
-    border: none;
+@mixin iui-table-header { 
+  user-select: none;
+  @include themed {
+    background-color: t(iui-color-background-3);
   }
 
-  .iui-tables-row {
-    margin-top: -1px;
+  .iui-row {
     display: flex;
     flex-grow: 1;
-    width: 100%;
-    border-top: 1px solid transparent;
-    border-left: 1px solid transparent;
-    border-right: 1px solid transparent;
-
-    @include themed {
-      border-bottom: solid 1px t(iui-color-background-4);
-    }
+  }
 
-    > .iui-tables-slot > .iui-tables-menu {
-      visibility: hidden;
-    }
+  .iui-cell:not(.iui-slot) {
+    @include iui-focus;
+    cursor: pointer;
+    position: relative;
+    overflow: visible;
 
-    &:hover:not(.iui-tables-row-active, .iui-tables-paginator) {
-      @include themed {
-        background-color: rgba(
-          t(iui-color-foreground-primary-rgb),
-          t(iui-opacity-6)
-        ) !important;
-      }
+    > .iui-borderless {
+      margin-left: $iui-xs;
 
-      > .iui-tables-slot > .iui-tables-menu {
-        visibility: visible;
+      &:not(.iui-active) {
+        visibility: hidden;
       }
     }
 
-    > .iui-tables-cell {
-      align-items: center;
-      display: flex;
-      flex-grow: 1;
-      min-width: 145px;
-      padding: $iui-table-density;
-      overflow: hidden;
-      list-style: none;
-      flex-basis: 145px;
-
-      > svg {
-        width: $iui-icons-default;
-        height: $iui-icons-default;
-        margin-right: $iui-sm;
+    > .iui-resizer {
+      height: 100%;
+      width: $iui-m;
+      position: absolute;
+      top: 0;
+      right: 0;
+      transform: translateX(50%);
+      touch-action: none;
+      cursor: ew-resize;
+      z-index: 100;
+      display: none;
+        
+      > .iui-resizer-bar {
+        height: 100%;
+        width: $iui-xxs;
```

Comment:
Whats the point to have this width and change it on hover?
I mean, with this width, element is not even shown (display: none).
So maybe we could set this width straight away to the correct one and no need to change it on hover.....?

Reply:
There is no transition applied on width..
Also, not sure it would even work, when you have display: none...

Reply:
oh ok, I tried with big width, there is something.. We cna leave it I guess, but barely noticable.

Reply:
It work. I did test it and it's cute animation. Better than on/off. I did change the size to be huge so you can see the transition.
![ResizerExageratedSize](https://user-images.githubusercontent.com/4070713/121216812-3f020280-c84f-11eb-9cbf-a8455922019e.gif)